logging/LogHelper_BaseLogRoute.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Craft;
+
+/**
+ * Log Helper Plugin.
+ *
+ * Advanced logging options
+ *
+ * @author    Nerds & Company
+ * @copyright Copyright (c) 2016, Nerds & Company
+ *
+ * @link      https://www.nerds.company
+ */
+abstract class LogHelper_BaseLogRoute extends \CLogRoute
+{
+    /**
+     * Formats a log message given different fields.
+     *
+     * @param array $log The log array.
+     *
+     * @return string The formatted message.
+     */
+    protected function formatLogMessage($log)
```

Comment:
I think it would be more compact if you'd move this function as `formatMessage` to `LogHelperPlugin::formatMessage`, then use it like that.